### ISSUE TITLE
QrPainter has no accessibility support (screen readers)

### ISSUE THREAD

**Issue by nilsreichardt** [theyakka/qr.flutter#101]
## Description
At the moment `QrPainter` can't be detected by screen readers... I already fixed it for `QrImage` (https://github.com/lukef/qr.flutter/pull/93), but `QrPainter` is missing. 

## Demo
You can see that Talkback (Android screen reader) can't focus the qr code widget.
 
![QrPainter](https://user-images.githubusercontent.com/24459435/87852330-43ed5900-c901-11ea-92f3-5fc5ce5071c7.gif)

## Steps to reproduce

<details>
<summary>Code to reproduce</summary>

```
import 'package:flutter/material.dart';
import 'package:qr_flutter/qr_flutter.dart';

void main() => runApp(MaterialApp(home: MainScreen()));

class MainScreen extends StatefulWidget {
  @override
  _MainScreenState createState() => _MainScreenState();
}

class _MainScreenState extends State<MainScreen> {
  @override
  Widget build(BuildContext context) {
    final message =
        'Hey this is a QR code. Change this value in the main_screen.dart file.';

    return Material(
      color: Colors.white,
      child: SafeArea(
        top: true,
        bottom: true,
        child: Container(
          child: Column(
            children: <Widget>[
              Expanded(
                child: Center(
                  child: Container(
                    width: 280,
                    child: CustomPaint(
                      size: Size.square(200),
                      painter: QrPainter(
                        data: message,
                        version: QrVersions.auto,
                      ),
                    ),
                  ),
                ),
              ),
              Padding(
                padding: EdgeInsets.symmetric(vertical: 20, horizontal: 40)
                    .copyWith(bottom: 40),
                child: Text(message),
              ),
            ],
          ),
        ),
      ),
    );
  }
}

```
</details>

1. Copy code to reproduce
2. Run app
3. Enable a screen reader (like Talkback on Android)
4. Try to focus the qr code widget with a screen reader --> impossible

**Comment by lukef** [comment]
Painter is pretty low level. The intention is for it to be consumed by higher level widgets which, I assume, would take care of this at that level. However, if you think there is a good use case for it I can look at it. The issue will be that QrImage consumes the painter directly so we probably would just switch the semantics to encompass the painter and ignore the image. Right?

**Comment by AndroidNils** [comment]
Yes, if it's possible to use `Semantics` at such low level.

**Comment by renansa27** [comment]
The QrPainter can't be detected by screen readers because it doesn't have a border, and if we want to get a qrCode as an Image we should use the .toImage() method but QrImage can't use .toImage() method only QrPainter. Could you guys fix it? Just add an option to add a border in QrPainter. Tnks

**Comment by lukef** [comment]
Why wouldn't you just add it into widget that has a border? Wouldn't that solve the issue?

**Comment by lukepighetti** [comment]
We're going to recommend that you tag this with your own `Semantics`. Closing this PR because there is nothing actionable for us.

**Comment by nilsreichardt** [comment]
@lukepighetti This an option, but if we decide to choose this option this should better documented (e.g. a headline "Accessibility" in the README.md with the hint to use your own `Semantics` to support screen reader). I will open a new issue.

**Comment by nilsreichardt** [comment]
@lukepighetti The new issue: #132
